packages/webapp/pages/onboarding.tsx

```diff
@@ -179,13 +190,38 @@ export function OnboardPage(): ReactElement {
     router.replace(getPathnameWithQuery(webappUrl, window.location.search));
   };
 
+  const isFeedSettingsDefined = useMemo(() => !!feedSettings, [feedSettings]);
+
+  const updateSettingsBasedOnExperience = useCallback(() => {
+    if (isSeniorUser && isFeedSettingsDefined) {
+      shouldUpdateAdvancedSettings.current = false;
+      updateAdvancedSettings({
+        advancedSettings: [
+          { id: 5, enabled: false },
+          { id: 10, enabled: false },
```

Comment:
Magic number here, maybe we can use a different mapping rather?

Reply:
Heh yeah, it just that we rarely seem to change these numbers, and since we depend on them to actually do the updates probably ok to do this. 
I can get these from the titles "Listicles" and "Memes" if u think is best, just thought either way doesn't make much difference since in this case, if we were to change the titles just a little bit then it would still mess up.

Reply:
Nah maybe just define them as `const LISTICLE_ADVANCED_SETTINGS_ID = 10` (just a bit clearer for others).

Reply:
Yup makes sense, in a moment